plugins/callback/foreman.py

```diff
@@ -210,13 +232,15 @@ def _ssl_verify(self, option):
 
         return verify
 
-    def _send_data(self, endpoint, host, data):
-        if endpoint == 'facts':
+    def _send_data(self, report_type, endpoint, host, data):
```

Comment:
in the rest of the code, `report_type` is `foreman` or `proxy`, but here it's `facts` or `report`.

how about this?
```suggestion
    def _send_data(self, data_type, report_type, host, data):
```

Reply:
Makes sense, I blindly renamed the attribute.